### DISCUSSION TITLE
Reading Elekta Synergy MLCi2 Log Files

### DISCUSSION THREAD

**Discussion by thanhtaiphys** [jrkerns/pylinac#518] [Q&A]
Dear all:

I hope this message finds you well. I am currently working with an **Elekta Synergy MLCi2 system**, and I need some assistance regarding the Elekta log files.

My system only exports log files in the ._dat format_, whereas I have learned that some other Elekta models can export log files to the _.trf format_, which can be read and analyzed using the **pylinac library.**

I would greatly appreciate any support or guidance on the following:

(1) How to read .dat files using pylinac or any other tools/methods that might work. @jrkerns 

(2) Is there a way to convert _.dat files to .trf files_ for analysis.

If anyone has experience dealing with this issue or knows of a solution, I would be very grateful for your insights.

Thank you in advance for your help!

Best regards,
Duong

**Comment by mchamberland** [comment]
This might be possible with [pymedphys](https://github.com/pymedphys/pymedphys)... I know it works for Agility MLC.

**Comment by thanhtaiphys** [comment]
Hi Marc @mchamberland 

What kind of logfile you are using? As I know, Agility MLC can export to  _.trf files_. We don't have issue with  .trf files

**Reply by mchamberland** [reply]
Hmm... it's been a long time. Let me check.

**Reply by mchamberland** [reply]
Ah, indeed looks like only .trf format. 

Then, maybe look into RadCalc. It has an Elekta MLC module. Not sure what format it supports. 

I know it connects directly to the iCom so it can monitor delivery. Not sure about retrospective log file analysis.

**Reply by thanhtaiphys** [reply]
Yes, Thank you!
I did a quick Google search, and it seems like everyone primarily works with the _.trf_ format for Elekta logs. It looks like using the _.dat_ format is quite uncommon.

**Comment by thanhtaiphys** [comment]
We have been following the same export procedures for both the Elekta Agility MLC and the Elekta Synergy MLCi2. However, while the Agility MLC exports the log files in the **.trf** format, the Synergy MLCi2 outputs them as **.dat** files. Therefore, we are unsure how to read these **.dat file**s or if there is a way to export or convert the .dat files to .trf format for analysis with pylinac
